src/components/Home/blocks/Activities/Activities.tsx

```diff
@@ -0,0 +1,31 @@
+import { get, size } from "lodash";
+import { Title } from "@deskpro/app-sdk";
+import { NoFound } from "../../../common";
+import { isLast } from "../../../../utils";
+import { ActivityItem } from "./ActivityItem";
+import type { FC } from "react";
+import type { Activity } from "../../../../services/sellsy/types";
+
+export type Props = {
+  activities: Activity[];
+};
+
+const Activities: FC<Props> = ({ activities }) => {
+  return (
+    <>
+      <Title title={`Activities (${size(activities)})`}/>
+      {(!Array.isArray(activities) || !size(activities))
```

Comment:
Minor: Does `Array.isArray(activities)` add any value given the `Props` say this is always an `array`, or should the types be updated to say this is maybe `nullable`?

Reply:
Not sure I agree with this logic. If you're saying that `activities` could be something other than an `array` then we should add that to the type. If it never happens, then we don't need this check.

I think leaving it as-is causes confusion, especially when using linters and code quality tools